public/static/global.css

```diff
@@ -347,115 +348,9 @@ button:focus {
   border-color: #666;
 }
 
-/* APP LAYOUT STYLES */
-
-.app-layout {
-  display: grid;
-  min-height: 100vh;
-  /* align-items: stretch; */
-  /* grid-template-rows: max-content max-content; */
-}
-
-.app-layout--vertical {
-  grid-template-rows: max-content auto;
-}
-
-.app-layout--horizontal {
-  grid-template-columns: var(--drawer-width) auto;
-}
-
-.toolbar {
-  align-self: start;
-  width: 100%;
-  height: 100%;
-  min-height: var(--header-height);
-  display: grid;
-  z-index: 100;
-  background-color: var(--toolbar-background-color);
-  color: var(--toolbar-text-color);
-}
-
-.toolbar--horizontal {
-  box-sizing: border-box;
-  align-content: start;
-  padding-top: var(--space-2x);
-  padding-bottom: var(--space-2x);
-  align-items: start;
-  justify-items: end;
-  grid-auto-flow: row;
-  grid-row-gap: var(--space-2x);
-}
-
-.toolbar--vertical {
-  grid-auto-flow: column;
-  grid-column-gap: var(--space-2x);
-}
-
-.toolbar--sticky {
-  position: sticky;
-  top: 0;
-}
-
-.content {
-  align-self: stretch;
-  --header-size: minmax(var(--increment), max-content);
-  --footer-size: minmax(var(--increment), max-content);
-  --body-width: var(--main-content-width);
-
-  display: grid;
-  align-items: start;
-  height: 100%;
-  grid-template-columns:
-    [left] minmax(0px, max-content)
-    [body-content] auto
-    [right] minmax(0px, max-content);
-  grid-template-rows:
-    [body-header] var(--header-size)
-    [body-content] auto
-    [body-footer] var(--footer-size);
-}
-
-.content--no-header {
-  --header-size: 0px;
-}
-.content--no-footer {
-  --header-size: 0px;
-}
-
-.content--centered-body {
-  grid-template-columns:
-    [left] auto
-    [left-gutter] minmax(0px, 80px)
-    [body-content] var(--body-width)
-    [right-gutter] minmax(0px, 80px)
-    [right] auto;
-}
-
-.content-header {
-  grid-column: body-content;
-  grid-row: body-header;
-  display: grid;
-  align-items: center;
-  height: 100%;
-  grid-auto-flow: column;
-  align-items: center;
-  justify-items: start;
-}
-.content-content {
-  grid-column: body-content;
-  grid-row: body-content;
-  align-self: stretch;
-}
-.content-footer {
-  grid-column: body-content;
-  grid-row: body-footer;
-  display: grid;
-  align-items: center;
-  height: 100%;
-}
-
 /* surfaces are the main thing */
 /* needs depth and roundedness as the main concerns */
+/* davor note: these are used in stories only, they might get moved */
```

Comment:
we shoudl add a GH issue around updating storybook to have the same setup as graph-paper (including custom CSS)